raku-mode.el

```diff
@@ -49,6 +49,7 @@
     (define-key map (kbd "C-c C-c") 'raku-send-line-to-repl)
     (define-key map (kbd "C-c C-r") 'raku-send-region-to-repl)
     (define-key map (kbd "C-c C-h") 'raku-send-buffer-to-repl)
+    (define-key map (kbd "C-c C-m") 'raku-reload-repl-with-module)
```

Comment:
Doesn't Emacs translate C-m to Return (that being ^M)?

The binding doesn't seem to work for me in any case.

Reply:
`(kbd "…C-m…")` will never work except on a GUI, and won’t even work on certain GUI’s that present <kbd>C-m</kbd> and <kbd>⏎</kbd> identically to Emacs.

I’d agree this needs a different binding.